tests/api/core/strapi/api/sort/sort.test.api.ts

```diff
@@ -211,6 +211,17 @@ describe('Sort', () => {
     ]);
   });
 
+  test('Sort on invalid key returns ValidationError', async () => {
```

Comment:
Note: this was a very false start on where to put the tests. It's a case we don't actually have a test for, but it actually belong in tests/api/core/strapi/api/validate/validate-query.test.api.js so I'll move it before opening the PR